amundsen_application/api/utils/data_utils.py

```diff
@@ -0,0 +1,29 @@
+from typing import Dict
+
+from amundsen_application.api.utils.request_utils import get_query_param
+
+
+def get_table_key(args: Dict) -> str:
```

Comment:
docstring. Since we put the function in a util file, others should be able to leverage it.